test/hotspot/jtreg/compiler/inlining/PrintInlining.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2015, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Not sure if the copyright should really mention 2015 :)

Reply:
Good catch!

I copied the header and *specially* updated the second copyright year (but forgot that we only need one for a new file :))